admin_tests/pre_reg/test_views.py

```diff
@@ -369,6 +376,82 @@ def test_checkin_files(self):
         for q, f in self.d_of_qs.iteritems():
             nt.assert_equal(None, f.checkout)
 
+    def test_approved_does_not_checkout_files(self):
```

Comment:
So in the vein of "fewer tests make the testing go faster", would it make sense to make one or two tests that take a file through the various states and checks at each of those states rather than having all the different tests?